README.md

```diff
@@ -160,6 +160,26 @@ pytest tests/test_basics.py # Ces test peuvent prendre jusqu'à 60 secondes.
 ```
 :tada: OpenFisca-France est prêt à être utilisé !
 
+### Installez un environnement virtuel avec conda
+
+Nous conseillons cette procédure pour les personnes utilisant Windows et n'ayant pas d'environnement Python fonctionnel. Elle fonctionne également sous Linux et macOS.
+
+Ceci vous permet d'obtenir en une seule installation :
+- Python
+- Le gestionnaire de paquets [Anaconda.org](https://docs.anaconda.com/anacondaorg/user-guide/)
+- Le gestionnaire d'environnement Python virtuel : [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
+
+Voici les étapes à suivre : 
+
+- Installer la version communautaire/gratuite en suivant la procédure décrite sur [le site Anaconda](https://www.anaconda.com/products/individual). A noter que Anaconda occupe beaucoup d'espace disque (>2 Go), vous pouvez installer à la place [Miniconda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html) qui occupe beaucoup moins d'espace disque. Cependant vous n'aurez pas l'interface graphique [Anaconda Navigator](https://docs.anaconda.com/anaconda/navigator/index.html) qui vous permet de gérer vos environnements. Si vous êtes habituée à gérer les choses en ligne de commande, préfèrer Miniconda.
+- Depuis le menu démarrer, exécuter `Anaconda Powershell Prompt`.
+- Exécuter les commandes suivantes dans le shell:
+  - Créer un environnement virtuel dédié : `conda create --name openfisca python=3.7`
+  - Activer l'environnement : `conda activate openfisca`
```

Comment:
> Cette commande, à sa première exécution (Miniconda), m'a renvoyé cette erreur : 
```shell
conda activate openfisca

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell
(...)
```

> Puis `conda init bash` s'est achevé avec : 
`==> For changes to take effect, close and re-open your current shell. <==`
Qui semble avoir activé un environnement par défaut : 
`(base) [sacha@home conda-test]$ `